source/extensions/common/aws/credentials_provider_impl.cc

```diff
@@ -425,7 +426,7 @@ void InstanceProfileCredentialsProvider::onMetadataError(Failure reason) {
   }
 }
 
-TaskRoleCredentialsProvider::TaskRoleCredentialsProvider(
+ContainerRoleCredentialsProvider::ContainerRoleCredentialsProvider(
```

Comment:
How about just renaming it to `ContainerCredentialsProvider` based on 
https://docs.aws.amazon.com/sdkref/latest/guide/feature-container-credentials.html ?

Reply:
ack